Corona-Warn-App/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,1007 @@
+
+    <!-- XHED: Page title for menu information page -->
+    <string name="information_title">"Uygulama Bilgileri"</string>
+    <!-- XTXT: Version information text -->
+    <string name="information_version">"Versiyon: %1s"</string>
+    <!-- XHED: Page title for about information page, also menu item / button text -->
+    <string name="information_about_title">"Uygulama Hakkında"</string>
+    <!-- XHED: Subtitle for about information page -->
+    <string name="information_about_headline">"Koronavirüs ile birlikte mücadele edelim"</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body_emphasized">"Robert Koch Institute (RKI), Almanya Federal Cumhuriyeti\'nde kamu sağlığı alanında merkezi kurumdur. RKI, federal hükûmet için Corona-Warn-App uygulamasını yayınlamaktadır. Uygulama, daha önce açıklanan kamu sağlığı önlemlerine ilişkin dijital bir uzantı niteliğindedir: sosyal mesafe, hijyen uygulamaları ve yüz maskeleri."</string>
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body">"Uygulamayı kullanan herkes, enfeksiyon zincirlerinin takip edilmesine ve kırılmasına yardımcı olur. Uygulama, diğer kişilerle karşılaşmaları cihazınızda yerel olarak kaydeder. Daha sonra COVID-19 tanısı konan kişilerle karşılaşmışsanız size bildirim gönderilir. Kimliğiniz ve gizliliğiniz daima koruma altındadır."</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_about_illustration_description">"Bölgedeki bir grup insan cihazlarını kullanıyor."</string>
+    <!-- XHED: Page title for privacy information page, also menu item / button text -->
+    <string name="information_privacy_title">"Veri Gizliliği Bilgileri"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"Bir kadın cihazında Corona-Warn-App\'i kullanıyor. Kılıftaki kilit, şifrelenmiş verilerin sembolüdür."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_en.html"</string>
+    <!-- XHED: Page title for terms of use information page, also menu item / button text -->
+    <string name="information_terms_title">"Kullanım Koşulları"</string>
+    <!-- XHED: Page headline for terms of use information page -->
+    <string name="information_terms_headline">"İçerik"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"Bir kişi ekranında uzun bir metin olan bir cihaz tutuyor. Metnin yanında kullanım koşullarını kabul ettiğini belirten bir sembol olarak tik işareti bulunuyor."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_en.html"</string>
```

Comment:
should be "translated" to `terms_tr.html` to be consistent with the other languages, for now I added the english versions to this files.

Reply:
implemented